routes/org/repos.ts

```diff
@@ -779,7 +779,7 @@ async function triggerRenameNotification(
   output: ITemporaryCommandOutput[]
 ): Promise<void> {
   const { config, insights, operations, mailAddressProvider, viewServices } = providers;
-  insights.trackMetric({ name: 'RenameDefaultBranchs', value: 1 });
+  insights.trackMetric({ name: 'RenameDefaultBranches', value: 1 });
```

Comment:
Not sure if this will skew the existing metrics

Reply:
Nice! This shipped way before GitHub implemented native branch renaming, and at least on the Microsoft side, we don't encourage use of this feature anymore, so it's fine from my side!

On the basis of removing complexity, might make sense to cut the rename default branch code entirely.

Opened https://github.com/microsoft/opensource-management-portal/issues/415

Reply:
No worries, if you want to trim out that code, I can rebase this afterwards.

Reply:
I'll take this as-is for now, thanks!